### DISCUSSION TITLE
How to do Aura sync Mouse and Laptop Backlight ?

### DISCUSSION THREAD

**Discussion by InfiKnight-TGP** [seerge/g-helper#1991] [Q&A]
I want to match my laptop backlight and mouse light pls help

**Comment by seerge** [comment]
@InfiKnight-TGP hello, if you mouse us listed and visible in G-Helper, you can just set same backlight modes / colors for both Laptop Keyboard and a mouse :)

**Reply by Danila0501** [reply]
Hi! It’s not a problem when you choose one color or a rainbow (since there are so many colors), but when you try to choose a color cycle, it doesn’t stay synchronized. I think if you make this function work like Aura Sync, it would be awesome!
